content/docs/2.0/scalers/mysql.md

```diff
@@ -33,11 +33,11 @@ You can authenticate by using connection string or password authentication.
 
 **Connection String Authentication:**
 
-- `connection` - Connection string for MySQL database
+- `connectionFromEnv` - Connection string for MySQL database
 
 **Password Authentication:**
 
-- `connection` - Password for configured user to login to MySQL database
+- `connectionFromEnv` - Password for configured user to login to MySQL database
 variables.
 
```

Comment:
Can we update below please?

Reply:
the example below is fine as is because it's using a `TriggerAuthentication` which hasn't changed.